<?xml version="1.0"?>
<launch>
    <!-- Decision Node -->
    <node pkg="decision" type="decision_node" name="decision_node" output="screen">
        <!-- 参数设置 -->
        <param name="full_hp_threshold" type="double" value="95.0" />
        <param name="low_hp_threshold" type="double" value="10.0" />
        <param name="enable_spin_at_base" type="bool" value="true" />
        <param name="enable_stop_spin_when_low_hp" type="bool" value="true" />
        <param name="position_tolerance" type="double" value="0.25" />
        <param name="orientation_tolerance" type="double" value="4" />
    </node>
</launch>